modules/govuk_jenkins/templates/jobs/check_content_consistency.yaml.erb

```diff
@@ -0,0 +1,91 @@
+---
+- scm:
+    name: check-content-consistency
+    scm:
+      - git:
+          url: git@github.gds:gds/router-data.git
+          basedir: router-data
+          branches:
+            - master
```

Comment:
Yes, we use it to include routes in the CSV files that aren't in the router dump.